Packs/SecureWorks/ReleaseNotes/4_1_0.md

```diff
@@ -0,0 +1,15 @@
+
+#### Integrations
+##### TaegisXDR
+- Added the following new commands:
+    - ***taegisxdr-create-comment***
+    - ***taegisxdr-fetch-comment***
+    - ***taegisxdr-fetch-comments***
+    - ***taegisxdr-update-comment***
+- Added ***url*** field output for reference in most commands.
+- Added test for test-module command.
+- Added errors when an investigation or playbook execution cannot be found.
+- Replaced the readable output of all commands with Markdown tables.
+- Updated documentation related to Context Data of all commands.
+    - Context path was updated in 4.0.0 but the documentation had not been updated to reflect the changes.
```

Comment:
```suggestion
```
I don't think it needs to be written in the release notes

Reply:
Sure, I can remove that once we confirm we're okay with the previous comment.